package.json

```diff
@@ -55,22 +55,22 @@
   },
   "peerDependencies": {
     "@warp-ds/css": "2.0.1",
-    "react": "18.x"
+    "@lingui/core": "5.x",
+    "react": "18.x || 19.x"
   },
   "scripts": {
-    "build-storybook": "storybook build && touch ./storybook-static/.nojekyll",
+    "build-storybook": "lingui compile && storybook build && touch ./storybook-static/.nojekyll",
     "build:clean": "rm -rf dist",
     "build:eik": "node esbuild.js 17 && node esbuild.js 18",
     "build:npm": "node build.js",
     "build:types": "tsc && sed 's+declare module \"index\"+declare module \"@warp-ds/react\"+g' ./dist/npm/index.d.ts > ./dist/npm/updated.d.ts && mv ./dist/npm/updated.d.ts ./dist/npm/index.d.ts",
-    "build": "pnpm run build:clean && pnpm run build:npm && pnpm run build:eik && pnpm run build:types",
+    "build": "pnpm run build:clean && lingui compile && pnpm run build:npm && pnpm run build:eik && pnpm run build:types",
     "commit": "cz",
     "dev": "storybook dev -p 9003 --ci",
     "lint:check": "eslint . --ext ts,tsx,js,jsx,cjs,mjs --ignore-path .gitignore",
     "lint": "eslint . --fix --ext ts,tsx,js,jsx,cjs,mjs --ignore-path .gitignore",
     "messages:compile": "lingui compile",
     "messages:extract": "lingui extract",
-    "postinstall": "lingui compile",
```

Comment:
Removed the postinstall step as it wasn't working in the other repo with the teams using our package.